src/open_inwoner/openzaak/tests/test_case_detail.py

```diff
@@ -456,8 +533,8 @@ def setUp(self):
                     "info_id": self.informatie_object["uuid"],
                 },
             ),
-            created=datetime.datetime.fromisoformat(
-                self.zaak_informatie_object["registratiedatum"]
+            created=dateutil.parser.parse(
+                self.zaak_informatie_object_old["registratiedatum"]
```

Comment:
Note for other reviewers: we discussed this and `dateutil.parser.parse` is acceptable here because the zgw_consumers `factory()` also uses that and it produces different timezones then `datetime.fromisoformat` so the equality check failed. 

There might be a way to convert it but let's declare that out of scope for this PR because it is a hassle.